pkg/build/api/validation/validation.go

```diff
@@ -4,8 +4,11 @@ import (
 	"net/url"
 
 	errs "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/validation"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
+	image "github.com/openshift/origin/pkg/image/api"
```

Comment:
I'm using image consistently in all the image code.  With builds it's a bit trickier.  I can make it imageapi when it's used at the same time as buildapi.

----- Original Message -----
> >  
> >  	buildapi "github.com/openshift/origin/pkg/build/api"
> > +	image "github.com/openshift/origin/pkg/image/api"
> 
> i prefer imageapi personally, so i'd rather change this one than those.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/679/files#r23208223